src/AssignmentsStorageInterface.php

```diff
@@ -7,6 +7,12 @@
 /**
  * `AssignmentsStorageInterface` represents a storage for assignment of RBAC items (a role or a permission) to a user
  * used in {@see Manager}.
+ *
+ * @psalm-type RawAssignment = array{
```

Comment:
Seems, it DB-specifc type. In other implementation type can be other. Right?

Reply:
Yes. Removed `RawAssignment` and `RawItem`.